frontend/src/layout/layout.component.tsx

```diff
@@ -2,11 +2,17 @@ import React from 'react';
 
 import { NavBar } from './navbar';
 
-export const Layout: React.FC<{}> = ({ children }) => {
+interface Props {
```

Comment:
Let's move this into the interfaces folder. Try to follow the guidelines we are already using in that folder.

Reply:
I´ve moved the interface to a file inside the interfaces folder.